docs/README.md

```diff
@@ -77,3 +169,100 @@ litexa test
 # deploy to aws
 litexa deploy
 ```
+</div>
+</div>
+
+
+
+
+<div class="faquestions">
+
+<FAQuestion>
+
+## How much does Litexa cost? Where does it come from?
+
+Litexa is *free!*
+
+Litexa is an Alexa Labs community-supported project (alpha) from the Alexa Games team at Amazon.
+We (Alexa Games) have used Litexa to develop and ship 20+ Alexa skills, YMMV.
```

Comment:
Would consider elaborating on YMMV as it's not a very common abbreviation and might be confusing to people. :)

Reply:
Good point. And here's me usually the one against abbreviations : )